File: sol/1832.cpp
struct Solution {
    inline bool checkIfPangram(string &sentence) {
        int flag = 0;
        short count = 0;
        for (char c : sentence){
            int m = 1 << (c - 'a');
            if ((flag & m) == 0){
                ++count;
                flag |= m;
            }
        }
        return count == 26;
    }
};

File: sol/1832/1832.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        occurred = [False] * 26
        count = 0
        for c in sentence:
            if not occurred[ord(c) - ord('a')]:
                count += 1
            occurred[ord(c) - ord('a')] = True
        return count == 26
